service/chain/disk_test.go

```diff
@@ -90,17 +91,10 @@ func TestDisk_Events(t *testing.T) {
 }
 
 func inMemoryDB(t *testing.T) *badger.DB {
```

Comment:
I would rename this function, given that it doesn't really create the DB anymore, but populates it.